libs/filabridge/include/private/filament/UibStructs.h

```diff
@@ -212,8 +213,8 @@ struct PerViewUib { // NOLINT(cppcoreguidelines-pro-type-member-init)
 };
 
 // 2 KiB == 128 float4s
-static_assert(sizeof(PerViewUib) == sizeof(math::float4) * 128,
-        "PerViewUib should be exactly 2KiB");
+static_assert(sizeof(PerViewUib) == sizeof(math::float4) * 144,
+        "PerViewUib should be 16 x 144 bytes");
```

Comment:
no this test should stay at 2KiB (see above). instead you need to change the '38' on line 212.